mentat/code_context.py

```diff
@@ -205,11 +190,9 @@ async def _get_code_message(
             ]
         code_message += ["Code Files:\n"]
         meta_tokens = count_tokens("\n".join(code_message), model)
+        remaining_tokens = max(0, max_tokens - meta_tokens)
 
-        auto_tokens = config.auto_tokens
-        remaining_tokens = max(0, min(auto_tokens, max_tokens - meta_tokens))
-
-        if remaining_tokens == 0:
+        if not config.auto_context or remaining_tokens <= 0:
```

Comment:
I like the `<=` comparison. It means we can remove the max from the `remaining_tokens` computation as well.